lib/beaker/host/windows.rb

```diff
@@ -46,6 +47,7 @@ def self.foss_defaults
         'puppetvardir'      => '`cygpath -smF 35`/PuppetLabs/puppet/var',
         'distmoduledir'     => '`cygpath -smF 35`/PuppetLabs/puppet/etc/modules',
         'sitemoduledir'     => 'C:/usr/share/puppet/modules',
+        'facter-factsd'     => '/cygdrive/c/ProgramData/PuppetLabs/facter/facts.d/',
```

Comment:
should that be something like ``'`cygpath -smF 35`/PuppetLabs/facter/facts.d/' `` ?